ElectronClient/app/gui/ResourceScreen.tsx

```diff
@@ -0,0 +1,180 @@
+import * as React from 'react';
+import { groupBy, orderBy } from 'lodash';
```

Comment:
Can we not use lodash? As I'd like to remove this package eventually. If necessary the required functions could be added to ArrayUtils or ObjectUtils.

Reply:
Just for my information, what's the intent behind removing lodash? To me it seems like an excellent collection of frequently used utilities. We can put them into Utils classes, but then we have to maintain them ourselves

Reply:
Sorting after multiple keys (for example size & name), as is possible with the current implementation, is certainly possible with Vanilla JS / some code block copied from somewhere, but I believe letting lodash handle the details is a bit easier to understand. But your call in the end!

Reply:
Hmm, I don't know. I remember last time I checked that lodash was quite huge in node_modules, all for relatively small utility functions, but maybe I'm mistaken. Let's leave it as it is for now then, and I'll review later.